### ISSUE TITLE
Error running the `onepub import` command on CI

### ISSUE THREAD

**Issue by gabrielbertollo** [onepub-dev/onepub#464]
**Describe the bug**
While running my CI for Codemagic, I'm getting an error with the onepub build process.

**To Reproduce**
1. Add the `flutter pub global activate onepub` and the `onepub import` commands to the codemagic.yaml file.
2. Add the ONEPUB_TOKEN to the environment.
3. Run the app build.

**Expected behavior**
I use the same commands in my GitHub CI and it is working with the same token.

** Problems with CLI tooling **
If your issue is with the OnePub CLI tooling then:

Run onepub doctor and post the output here:

```
OnePub version: 3.1.0

Platform
Dart version:   2.19.1
Dart path:      /Users/gabrielb/development/flutter

OS:             macos
OS version:     Version 13.1 (Build 22C65)

Locale:         en-BR
Path separator: /


URLs
Web site:       https://onepub.dev/
API endpoint:   https://onepub.dev/api/xpdpsdavuh/

Environment
Pub Cache:      /<HOME>/.pub-cache
PUB_CACHE:      not set and not used.
PATH:
                /opt/homebrew/bin
                /opt/homebrew/sbin
                /usr/local/bin
                /System/Cryptexes/App/usr/bin
                /usr/bin
                /bin
                /usr/sbin
                /sbin
                /Library/Apple/usr/bin
                /<HOME>/development/flutter/bin
                /<HOME>/.pub-cache/bin

Shell Settings
$SHELL:         /bin/zsh
Detected:       zsh
Start script:   /<HOME>/.zshrc

Repository tokens
https://onepub.dev/api/advdavuhno/
https://onepub.dev/api/xpdpsdavuh/

Status
Logged In:      true
Active Member:  code@prefeituradofuturo.com.br
Organisation:   code@prefeituradofuturo.com.br
checking status...
OnePub: status healthy.

Server Version: 3.96.14
```

**Additional context**
<details>
<summary>Here is the full error log for inspection:</summary>

Failed to build onepub:onepub:
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:723:40: Error: Field '_unique___locales_item_0' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_0;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:724:40: Error: Field '_unique___locales_item_1' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_1;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:725:40: Error: Field '_unique___locales_item_2' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_2;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:726:40: Error: Field '_unique___locales_item_3' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_3;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:727:40: Error: Field '_unique___locales_item_4' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_4;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:728:40: Error: Field '_unique___locales_item_5' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_5;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:729:40: Error: Field '_unique___locales_item_6' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_6;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:730:40: Error: Field '_unique___locales_item_7' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_7;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:731:40: Error: Field '_unique___locales_item_8' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_8;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:732:40: Error: Field '_unique___locales_item_9' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_9;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:733:40: Error: Field '_unique___locales_item_10' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_10;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:734:40: Error: Field '_unique___locales_item_11' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_11;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:735:40: Error: Field '_unique___locales_item_12' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_12;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:741:35: Error: Field '_unique___names_item_0' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_0;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:742:35: Error: Field '_unique___names_item_1' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_1;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:743:35: Error: Field '_unique___names_item_2' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_2;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:744:35: Error: Field '_unique___names_item_3' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_3;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:745:35: Error: Field '_unique___names_item_4' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_4;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:746:35: Error: Field '_unique___names_item_5' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_5;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:747:35: Error: Field '_unique___names_item_6' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_6;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:748:35: Error: Field '_unique___names_item_7' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_7;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:749:35: Error: Field '_unique___names_item_8' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_8;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:750:35: Error: Field '_unique___names_item_9' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_9;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:751:35: Error: Field '_unique___names_item_10' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_10;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:752:35: Error: Field '_unique___names_item_11' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_11;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:753:35: Error: Field '_unique___names_item_12' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_12;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:296:35: Error: Field 'gr_name' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? gr_name;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:299:35: Error: Field 'gr_passwd' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? gr_passwd;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:303:17: Error: Field 'gr_gid' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external int? gr_gid;
                ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:306:48: Error: Field 'gr_mem' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Pointer<ffi.Int8>>? gr_mem;
                                               ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:291:35: Error: Field 'pw_name' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_name;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:295:35: Error: Field 'pw_passwd' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_passwd;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:299:17: Error: Field 'pw_uid' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external int? pw_uid;
                ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:303:17: Error: Field 'pw_gid' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external int? pw_gid;
                ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:306:35: Error: Field 'pw_gecos' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_gecos;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:309:35: Error: Field 'pw_dir' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_dir;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:312:35: Error: Field 'pw_shell' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_shell;
                                  ^
Failed to build onepub:opub:
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:723:40: Error: Field '_unique___locales_item_0' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_0;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:724:40: Error: Field '_unique___locales_item_1' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_1;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:725:40: Error: Field '_unique___locales_item_2' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_2;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:726:40: Error: Field '_unique___locales_item_3' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_3;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:727:40: Error: Field '_unique___locales_item_4' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_4;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:728:40: Error: Field '_unique___locales_item_5' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_5;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:729:40: Error: Field '_unique___locales_item_6' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_6;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:730:40: Error: Field '_unique___locales_item_7' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_7;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:731:40: Error: Field '_unique___locales_item_8' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_8;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:732:40: Error: Field '_unique___locales_item_9' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_9;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:733:40: Error: Field '_unique___locales_item_10' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_10;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:734:40: Error: Field '_unique___locales_item_11' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_11;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:735:40: Error: Field '_unique___locales_item_12' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<__locale_data>? _unique___locales_item_12;
                                       ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:741:35: Error: Field '_unique___names_item_0' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_0;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:742:35: Error: Field '_unique___names_item_1' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_1;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:743:35: Error: Field '_unique___names_item_2' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_2;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:744:35: Error: Field '_unique___names_item_3' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_3;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:745:35: Error: Field '_unique___names_item_4' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_4;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:746:35: Error: Field '_unique___names_item_5' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_5;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:747:35: Error: Field '_unique___names_item_6' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_6;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:748:35: Error: Field '_unique___names_item_7' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_7;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:749:35: Error: Field '_unique___names_item_8' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_8;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:750:35: Error: Field '_unique___names_item_9' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_9;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:751:35: Error: Field '_unique___names_item_10' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_10;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:752:35: Error: Field '_unique___names_item_11' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_11;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/string/string.dart:753:35: Error: Field '_unique___names_item_12' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? _unique___names_item_12;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:296:35: Error: Field 'gr_name' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? gr_name;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:299:35: Error: Field 'gr_passwd' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? gr_passwd;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:303:17: Error: Field 'gr_gid' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external int? gr_gid;
                ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/grp.dart:306:48: Error: Field 'gr_mem' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Pointer<ffi.Int8>>? gr_mem;
                                               ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:291:35: Error: Field 'pw_name' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_name;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:295:35: Error: Field 'pw_passwd' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_passwd;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:299:17: Error: Field 'pw_uid' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external int? pw_uid;
                ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:303:17: Error: Field 'pw_gid' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external int? pw_gid;
                ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:306:35: Error: Field 'pw_gecos' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_gecos;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:309:35: Error: Field 'pw_dir' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_dir;
                                  ^
../.pub-cache/hosted/pub.dev/posix-2.2.3/lib/src/pwd.dart:312:35: Error: Field 'pw_shell' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.
  external ffi.Pointer<ffi.Int8>? pw_shell;
                                  ^
The Flutter CLI developer tool uses Google Analytics to report usage and diagnostic data
along with package dependencies, and crash reporting to send basic crash reports.
This data is used to help improve the Dart platform, Flutter framework, and related tools.

Telemetry is not sent on the very first run.
To disable reporting of telemetry, run this terminal command:

flutter --disable-telemetry.
If you opt out of telemetry, an opt-out event will be sent,
and then no further information will be sent.
This data is collected in accordance with the
Google Privacy Policy (https://policies.google.com/privacy).

Please note that analytics reporting was already disabled, and will continue to be disabled.

/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/build_script_5_1_bxm0thge: line 4: onepub: command not found
Resolving dependencies...
Because mais_cidades depends on app_core any which doesn't exist (authentication failed), version solving failed.

https://onepub.dev/api/xpdpsdavuh/ package repository requested authentication!
You can provide credentials using:
    dart pub token add https://onepub.dev/api/xpdpsdavuh/
Run 'onepub login' and try again or read https://onepub.dev/FAQ/3f883941-d670-4e12-9682-f0c5a3622829.

Build failed :|
Step 5 script `Get Flutter packages` exited with status code 69
</details>

**Comment by bsutton** [comment]
Thanks for the bug report.
The problem is in the dart posix library.

We have a fix in hand and expect to publish it Monday latest.

**Comment by bsutton** [comment]
gabriel,
so some good news.

The team have pushed a beta out.
We know there is still a problem with the `onepub pub add` command but the login/export/import process should now be working as expected.
Can you give the beta a run and let us know how it goes.
We continue to work through the remaining issues.

You can install the beta by running:

```
dart pub global activate onepub 4.0.0-beta.2
```

**Comment by gabrielbertollo** [comment]
Hey Brett,

Thanks for the quick response.

I changed the command in my CI process with the beta one you gave me, and it seems to be working fine now.
The process passed the `onepub import` command successfully.

**Comment by bsutton** [comment]
gabriel,
we have released onepub 4.0 which resolves all of the dart 3.x issues.

Can you please upgrade to the latest version and let us know if you have any problems.

I will close this issue but feel free to reopen it if you have any problems.

**Comment by gabrielbertollo** [comment]
Hey Brett,

Just upgraded to the last version and everything is working.

Thanks for the support.
